### ISSUE TITLE
Scan job endlessly lists QUEUED status

### ISSUE THREAD

**Issue by gisjedi** [ngageoint/scale#1691]
**Description**
Very inconsistent job status various places throughout the UI, but I believe this is ultimately an API / message handler problem.

**Reproduction Steps**
Steps to reproduce the problem:
1. Create an input and output workspace. I did the following:

POST `/api/v6/workspaces/`
```
{
    "name": "testing-host-input",
    "title": "Testing Host Input",
    "description": "Testing Host Input",
    "is_active": true,
    "configuration": {
        "broker": {
            "host_path": "/dfs/testing/input",
            "type": "host"
        }
    }
}
```

POST `/api/v6/workspaces/`
```
{
    "name": "testing-host-output",
    "title": "Testing Host Output",
    "description": "Testing Host Output",
    "is_active": true,
    "configuration": {
        "broker": {
            "host_path": "/dfs/testing/output",
            "type": "host"
        }
    }
}
```
2. Create a scan process. I used the following:

POST `/api/v6/scans/`
```
{
  "title": "Scan host input workspace",
  "description": "This is my Scan process for detecting my favorite local host workspace!",
  "configuration": {
    "version": "1.0",
    "workspace": "testing-host-input",
    "scanner": {
      "type": "dir"
    },
    "recursive": true,
    "files_to_ingest": [{
      "filename_regex": ".*"
    }]
  }
}
```

3. Triggered the scan from the create config:

POST `/api/v6/scans/1/process/?ingest=true`

4. Navigated to the Scan page in the UI or just hit API (GET `https://scale.omega.aisohio.net/api/v6/scans/9/`). This gives me a job status of `QUEUED`... this is after the job has executed.

5. If I look at the jobs page or just hit API (GET `https://scale.omega.aisohio.net/api/v6/jobs/31/`) it claims its latest execution is `RUNNING`. This can be seen in the response at `execution.status`.

6. Looking at the log it appears to be complete and ingests were launched(GET `https://scale.omega.aisohio.net/api/v6/job-executions/37/logs/combined/`):
```
2019-07-02T18:56:46.430665254+00:00: 2019-07-02 18:56:46 INFO [ingest.scan.scanners.scanner(225)] Removed 0 duplicates of pre-existing ingests.
2019-07-02T18:57:14.446451472+00:00: 2019-07-02 18:57:14 INFO [ingest.scan.scanners.scanner(111)] Processed 201 files during scan.
2019-07-02T18:57:14.446547767+00:00: 2019-07-02 18:57:14 INFO [ingest.management.commands.scale_scan(90)] Scanner has stopped running
2019-07-02T18:57:14.446564903+00:00: 2019-07-02 18:57:14 INFO [ingest.management.commands.scale_scan(92)] Command completed: scale_scan
```

**Expected behavior**
Once a scan is launched, I expected job status to be updated to `QUEUED`, `RUNNING` and then `COMPLETED`. Job execution is being updated to `RUNNING` but never `COMPLETED`.

**Screenshots**
Scan Details page:

![image](https://user-images.githubusercontent.com/4768496/60543001-43c4f000-9ce3-11e9-9250-416619c030d4.png)

Scan Job Details page:

![image](https://user-images.githubusercontent.com/4768496/60543024-4de6ee80-9ce3-11e9-9f80-f03a8bcf16a6.png)

**Version and Environment Details**
 - OS: OSX Mojave 10.14.5
 - Browser: Chromium 75.0.3770.100
 - Scale API:  v6.9.0-snapshot+1d15ca7e
 - Scale UI: v0.4.3

**Comment by emimaesmith** [comment]
Similar issue is happening with Strike jobs after they are queued from an ingest.

**Comment by emimaesmith** [comment]
This particular issue is likely corrected via #1677. Scans are not being queued via the UI at the moment due to the `/scans/process` endpoint not being hit after the Scan is created. Created issue https://github.com/ngageoint/scale-ui/issues/188 in the UI to resolve this.

**Comment by mikenholt** [comment]
I tested this several times in Omega cluster and observed the Scan job moving through the various states successfully. We can repopen if different behavior is noticed elsewhere.
